### PR TITLE
Update to Ubuntu 20.04

### LINKED ISSUE #1027: x11vnc High Cpu Usage after running a day
## To Reproduce
I have build a docker image locally from official image to change default vnc password , it works fine at the beginning , **but x11vnc process get  high cpu usage （100% ）after running a day**

here is the command to start x11vnc  :
`x11vnc -usepw -forever -shared -rfbport 5900 -display :99.0 -noxrecord`

## Environment
OS: centos 7
Docker-Selenium image version: standalone-chrome-debug:3.141.59-zirconium
Docker version: Docker version 19.03.7
Exact Docker command to start the containers (if using docker-compose, provide 
the docker-compose file as well):
`docker run -d --net=host  -v /dev/shm:/dev/shm selenium/standalone-chrome-debug:3.141.59-zirconium`

### PR DESCRIPTION
**Pull request by hbouhadji**
**Thanks for contributing to the Docker-Selenium project!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines, applied for this repository.
Avoid large PRs, help reviewers by making them as simple and short as possible.

### Description

Should fix #1027 
```
CONTAINER ID        NAME                CPU %               MEM USAGE / LIMIT     MEM %               NET I/O             BLOCK I/O           PIDS
92a5cf5e324a        chrome-1_1   360.47%             7.722GiB / 62.83GiB   12.29%              37.7GB / 2.2GB      157MB / 10.5GB      2361
ddb309e5efdf        chrome-3_1   267.94%             7.263GiB / 62.83GiB   11.56%              33.9GB / 2.03GB     160MB / 9.61GB      2240
a66d87e336e7        chrome-2_1   297.31%             7.579GiB / 62.83GiB   12.06%              34GB / 1.95GB       176MB / 9.43GB      2367
04344bd3953f        chrome-6_1   311.54%             6.772GiB / 62.83GiB   10.78%              28.5GB / 1.68GB     151MB / 8.57GB      2108
1650acda4ff5        chrome-5_1   264.54%             7.647GiB / 62.83GiB   12.17%              37.1GB / 2.18GB     173MB / 10.2GB      2369
91df234e8897        chrome-4_1   293.95%             7.334GiB / 62.83GiB   11.67%              36.4GB / 2.03GB     170MB / 9.58GB      2240
188e19fe8995        selenium-hub        7.33%               1.527GiB / 62.83GiB   2.43%               98.6GB / 102GB      81.1MB / 0B         154
```
image

As you can see on the screen, after a day, x11vnc is using a lot of cpu.

According to @mokitoo and the changelog of https://github.com/LibVNC/x11vnc/releases the 0.9.15 version of x11vnc should fixes some buffer overflow and memleak.

But on the default repo of bionic, there is only the 0.9.13-3 (https://packages.ubuntu.com/bionic/x11vnc) version on it. But on focal there is the latest one (0.9.16-3 https://packages.ubuntu.com/focal/x11vnc)

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [x] I have read the [contributing](https://selenium.dev/documentation/en/contributing/) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by hbouhadji** [comment]
it seems there is an issue with NodeOpera, I'm investigating

**Comment by hbouhadji** [comment]
For some reasons, libffmpeg.so is missing when installing opera on ubuntu 20.04 (tip: https://gist.github.com/WMP/8042f545ea1b6309a5c8d35a371a52c8)

File: NodeOpera/generate.sh [hunk 1]
```diff
@@ -7,6 +7,9 @@ echo "# !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!" > ./Dockerfile
 echo "# NOTE: DO *NOT* EDIT THIS FILE.  IT IS GENERATED." >> ./Dockerfile
 echo "# PLEASE UPDATE Dockerfile.txt INSTEAD OF THIS FILE" >> ./Dockerfile
 echo "# !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!" >> ./Dockerfile
+echo "FROM ubuntu:bionic AS builder-codecs-ffmpeg" >> ./Dockerfile
+echo "RUN apt-get update -qqy && apt-get -qqy install chromium-codecs-ffmpeg-extra" >> ./Dockerfile
+echo "" >> ./Dockerfile
```
Comment (diemol) [inline NodeOpera/generate.sh:12]: Why don't you add these lines to the `NodeOpera/Dockerfile.txt` right away?

**Review by diemol — CHANGES_REQUESTED** [review]

File: NodeOpera/generate.sh (hunk 1 above)
Comment (hbouhadji) [reply NodeOpera/generate.sh:12]: Because these lines need to be before `FROM ${NAMESPACE}/node-base:${VERSION} >> ./Dockerfile`

**Review by diemol — APPROVED** [review]
Thank you, @hbouhadji!

**Comment by JeffreyAtQ** [comment]
@diemol 
Hey there. so, it looks like this PR once merged and 3.141 was re-tagged it made our .net selenium grid set up we had in teamcity and AWS cloudformation stop working: we would get this error when trying to created the OpenQA.Selenium.Remote.RemoteWebDriver with a OpenQA.Selenium.Chrome.ChromeOptions parameter:

```
OpenQA.Selenium.WebDriverException : Unexpected error. <html>
<head><title>502 Bad Gateway</title></head>
<body>
<center><h1>502 Bad Gateway</h1></center>
</body>
</html>
```

We use just the selenium/hub image and the selenium/node-chrome in our grid setup.

Rolling back to 3.141.59-20201119 fixed all our issues imidiately, so it seems that something with the ubuntu update has possibly gone awry.

I know this is a probably a tough thing to troubleshoot without telling you everything about our set up but I'm down to give any details you might need upon request

**Comment by diemol** [comment]
`latest` is always re-tagged, and without any more details it is hard to know what the problem is. 

If you'd like, please create a new issue with all the required information filled out.

**Comment by JeffreyAtQ** [comment]
@diemol 
> `latest` is always re-tagged, and without any more details it is hard to know what the problem is.
> 
> If you'd like, please create a new issue with all the required information filled out.

would you consider this a new bug or a regression? I ask because the details asked for in regression issue are very scant in comparison

**Pull request merged**